extensions/ql-vscode/src/query-history.ts

```diff
@@ -736,6 +750,49 @@ export class QueryHistoryManager extends DisposableObject {
       }
     }
   }
+  
+  private warnNoEvalLog() {
+    void showAndLogWarningMessage('No evaluator log is available for this run. Perhaps it failed before evaluation, or you are running with a version of CodeQL before ' + CliVersionConstraint.CLI_VERSION_WITH_PER_QUERY_EVAL_LOG + '?');
+  }
+
+  async handleShowEvalLog(
+    singleItem: QueryHistoryInfo,
+    multiSelect: QueryHistoryInfo[]
+  ) {
+    const { finalSingleItem, finalMultiSelect } = this.determineSelection(singleItem, multiSelect);
+
+    // Only applicable to an individual local query
+    if (!this.assertSingleQuery(finalMultiSelect) || !finalSingleItem || finalSingleItem.t !== 'local') {
+      return;
```

Comment:
Should this situation also result in some kind of error message?

Reply:
Hmm, good question. I followed what @aeisenberg did above for other similar drop downs. Any particular reason you went with just a `return` rather than a more explicit error?